FastSimulation/ParticlePropagator/plugins/MagneticFieldMapESProducer.cc

```diff
@@ -1,30 +1,21 @@
 #include "FastSimulation/ParticlePropagator/plugins/MagneticFieldMapESProducer.h"
-#include "FastSimulation/TrackerSetup/interface/TrackerInteractionGeometryRecord.h"
```

Comment:
This is also needed, please restore

Reply:
They are included in the .h file. So they are indeed redundant here